demos/src/demo-line.js

```diff
@@ -247,16 +244,16 @@ function isInRange(d0, d1, d) {
 }
 
 // Show charts if container available
-if (d3Selection.select('.js-line-chart-container').node()) {
+if (select('.js-line-chart-container').node()) {
     createLineChart();
     createBrushChart();
     createLineChartWithSingleLine();
     createLineChartWithFixedHeight();
     createLoadingState();
 
     redrawCharts = function(){
-        d3Selection.selectAll('.line-chart').remove();
-        d3Selection.selectAll('.brush-chart').remove();
+        selectAll('.line-chart').remove();
```

Comment:
I was wondering , can't you do `selectAll('.line-chart, .brush-chart').remove()` ?

Reply:
I can! Good catch!